subsys/bluetooth/host/shell/bt.c

```diff
@@ -3352,6 +3387,43 @@ static int cmd_read_all_remote_features(const struct shell *sh, size_t argc, cha
 }
 #endif
 
+#if defined(CONFIG_BT_FRAME_SPACE_UPDATE)
+static int cmd_frame_space_update(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err = 0;
+
+	if (default_conn == NULL) {
+		shell_error(sh, "Conn handle error, at least one connection is required.");
+		return -ENOEXEC;
+	}
+
+	for (size_t argn = 1; argn < argc; argn++) {
+		(void)shell_strtoul(argv[argn], 10, &err);
+
+		if (err) {
+			shell_help(sh);
+			shell_error(sh, "Could not parse input number %d", argn);
+			return SHELL_CMD_HELP_PRINTED;
+		}
+	}
+
+	const struct bt_conn_le_frame_space_update_param params = {
+		.frame_space_min = shell_strtoul(argv[1], 10, &err),
+		.frame_space_max = shell_strtoul(argv[2], 10, &err),
+		.phys = shell_strtoul(argv[3], 10, &err),
+		.spacing_types = shell_strtoul(argv[4], 10, &err),
```

Comment:
Does this require the user to enter the bitmasks as base-10 numbers? It would be more intuitive for the user to enter them as hex, I would think.